### PR TITLE
Train val

### PR DESCRIPTION
**Pull request by shashankiyer**

### CODE REVIEW

**Review by sourcesync — CHANGES_REQUESTED** [review]
I was able to run everything with your instruction.  Can you please add a section in the README that basically is a step-by-step to seeing things run ?  Something like this:

1.  Get access to a system with a recent version of tensorflow ( v<XXX) and a GPU.  We used the ubuntu deep learning image (v=20) and a G3.xlarge available at AWS EC2.  Note, once you login you will need to run "conda install tensorflow."

2.  Clone the repo and follow the instructions above to download the cifar10 data and pretrained model.

3.  Start training by typing:  "python finetune.py" . This will eventually print message indicating training is occurring.

4.  Let it train for a while ( we recommend X epochs ).  You can watch the training via Tensorflow's visual tensorboard utility.
4A.  If you are using an amazon EC2 instance, you will need to create a "tunnel" using this command.
4B.  Now run tensorboard like so...
4C.  On your local machine, you can browse over to "localhost:6006" to view the interactive results

5.  At any point during training, you can stop it via Ctrl-C.  This is a good chance to get some accuracy numbers.  Run the command "python similarity_search.py".  This will report mAP ( medium average precision ) that is described nicely here ( link ).

6.  You can resume training by type "python finetune.py"
